core/src/com/unciv/ui/screens/worldscreen/unit/actions/UnitActionsPillage.kt

```diff
@@ -65,7 +65,10 @@ object UnitActionsPillage {
                 if (pillagingImprovement) { // only Improvements heal HP
                     var healAmount = 25f
                     for (unique in unit.civ.getMatchingUniques(UniqueType.PercentHealthFromPillaging)) {
-                            healAmount *= unique.params[0].toPercent()
+                        healAmount *= unique.params[0].toPercent()
+                    }
+                    for (unique in unit.getMatchingUniques(UniqueType.PercentHealthFromPillaging)) {
```

Comment:
Use civ=true on those function, to have this only once instead of twice

Reply:
How do you mean? Where do I put "= true"?

Reply:
unit.getmatchinguniques has an optional parameter called includeCivUniques or something

Reply:
Like this?